EAD-Session-3-Assignment/src/com/metacube/servlets/SearchStudentInformation.java

```diff
@@ -0,0 +1,131 @@
+package com.metacube.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import javax.management.Query;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class SearchStudent
+ */
+@WebServlet("/SearchStudentInformation")
+public class SearchStudentInformation extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public SearchStudentInformation() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		
+		PrintWriter out = response.getWriter();
+		ResultSet resultSet;
+		String firstName = request.getParameter("search_by_fname");
+		String lastName = request.getParameter("search_by_lname");
+		String student_class = request.getParameter("search_by_class");
+		if(student_class != null)
+		{
+			resultSet = displayResultAccordingToClass(student_class);
```

Comment:
Pass the resultSet data to a jsp page and display the table data in the jsp page.

Reply:
We were told not to use jsp in these assignments